.circleci/config.yml

```diff
@@ -446,7 +446,8 @@ jobs:
       TAG: circleci-nightly
     steps:
       - checkout
-      - run: make submodule-sync
```

Comment:
if you checkout, also submodule-sync, they must go in pair

Reply:
Fixed.